docs_outdoornav_user_manual/api/api_endpoints/api_endpoints.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: API Endpoints
+sidebar_label: API Endpoints
+sidebar_position: 2
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+The ROS1 Noetic API can be used to monitor and manage OutdoorNav Software. Details
+are available through the child pages.
+
+- [Platform API](./platform_api)
+- [Autonomy API](./autonomy_api)
+- [Mission Manager API](./mission_manager_api)
+- [Definitions](./definitions)

```

Comment:
all of these links were not working for me in my local build

Reply:
Confirmed with @jmastrangelo-cpr that this appears to be working ok now. I'll validate on the automated build after merging.